onmt/inputters/corpus.py

```diff
@@ -149,7 +149,9 @@ def load(self, offset=0, stride=1):
                     tline = tline.decode('utf-8')
                     example = {
                         'src': sline,
-                        'tgt': tline
+                        'tgt': tline,
+                        'src_original': sline,
+                        'tgt_original': tline
```

Comment:
This "original" naming scheme is not very explicit.

Reply:
`src_raw` and `tgt_raw`?

Reply:
Well, I just saw you use the naming "original" in some other functions, so let's keep it coherent.
Though, you might want to add a comment when adding these fields to explicitly say why we may need them.